src/microbe_stage/editor/CellEditorComponent.Callbacks.cs

```diff
@@ -294,8 +294,9 @@ private void DoOrganelleUpgradeAction(OrganelleUpgradeActionData data)
     {
         data.UpgradedOrganelle.Upgrades = data.NewUpgrades;
 
-        // Uncomment when upgrades can visually affect the cell
-        // UpdateAlreadyPlacedVisuals();
+        microbeVisualizationOrganellePositionsAreDirty = true;
+
+        OnOrganellesChanged();
 
         UpdateStats();
 
```

Comment:
The line for `StartAutoEvoPrediction` should be deleted as `OnOrganellesChanged` already calls that. This same thing is in the undo method.